### PR TITLE
Stop wagging tails on crit

### LINKED ISSUE #25272: Tails do not stop wagging even in death
## Description

SLL

### PR DESCRIPTION
**Pull request by ArchPigeon**
## About the PR
fixes  #25272 

## Why / Balance

Tails do stop wagging in death, but it was correctly mentioned that they do not stop wagging while in crit.  This PR fixes that.

## Technical details

removed a code block that only toggled wagging if the creature was dead on state change

## Media

https://github.com/space-wizards/space-station-14/assets/12686577/601ab857-a1a6-4471-b75c-ff420f7d773c

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
- fix: Tails now stop wagging on crit

### CODE REVIEW

**Comment by ViceEmargo** [comment]
GitHub question, but why am I seeing the Tritium flammable touch commits in this PR?

**Comment by ArchPigeon** [comment]
> GitHub question, but why am I seeing the Tritium flammable touch commits in this PR?

I accidentally commited a change for a different PR to master before I made this branch, so I had to revert the commit.

TL;DR because I'm a dumbass

**Comment by ViceEmargo** [comment]
> > GitHub question, but why am I seeing the Tritium flammable touch commits in this PR?
> 
> I accidentally commited a change for a different PR to master before I made this branch, so I had to revert the commit.
> 
> TL;DR because I'm a dumbass

Hahaha. Thank you for making the mistake so hopefully I can avoid it in the future.
